core/modules/qdisp/QueryRequest.h

```diff
@@ -38,12 +38,13 @@
 // Local headers
 #include "czar/Czar.h"
 #include "qdisp/JobQuery.h"
-#include "qdisp/LargeResultMgr.h"
+#include "QdispPool.h"
```

Comment:
I think we still prefer "qdisp/QdispPool.h" even for headers from the same module.

Reply:
IDE refactoring for file names is a bit like autocorrect